### PR TITLE
feat(rpc): Add better logging and handling of error messages

### PR DESCRIPTION
**Pull request by danadi7**
#### Which issue(s) does this PR fixes?:

Fixes ORD-

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] No console errors on web
- [ ] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

File: apps/api/src/address/AddressController.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { Controller, Get, Param } from "@nestjs/common";
+import { BitcoinService, OrdProvider } from "@ordzaar/rpcservices";
```
Comment (joundy) [inline apps/api/src/address/AddressController.ts:2]: update it to use the new package naming

File: apps/api/src/address/AddressModule.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Module } from "@nestjs/common";
+import { BitcoinModule, OrdModule } from "@ordzaar/rpcservices";
```
Comment (joundy) [inline apps/api/src/address/AddressModule.ts:2]: ^

File: apps/api/src/ordinals/OrdinalsController.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { Controller, Get, Param } from "@nestjs/common";
+import { BitcoinService, OrdProvider } from "@ordzaar/rpcservices";
```
Comment (joundy) [inline apps/api/src/ordinals/OrdinalsController.ts:2]: ^

File: apps/api/src/ordinals/OrdinalsModule.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { Module } from "@nestjs/common";
+import { BitcoinModule, OrdModule } from "@ordzaar/rpcservices";
```
Comment (joundy) [inline apps/api/src/ordinals/OrdinalsModule.ts:2]: ^

File: apps/api/src/transactions/TransactionsController.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import { Controller, Get, Param, Post } from "@nestjs/common";
+import { BitcoinService } from "@ordzaar/rpcservices";
```
Comment (joundy) [inline apps/api/src/transactions/TransactionsController.ts:2]: ^

File: apps/api/src/transactions/TransactionsModule.ts [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import { Module } from "@nestjs/common";
+import { BitcoinModule } from "@ordzaar/rpcservices";
```
Comment (joundy) [inline apps/api/src/transactions/TransactionsModule.ts:2]: ^

File: apps/api/src/address/AddressController.ts (hunk 1 above)
Comment (danadi7) [reply apps/api/src/address/AddressController.ts]: File removed. This should be on the base-api branch on the other PR.

File: apps/api/src/address/AddressModule.ts (hunk 2 above)
Comment (danadi7) [reply apps/api/src/address/AddressModule.ts]: File removed. This should be on the base-api branch on the other PR.

File: apps/api/src/ordinals/OrdinalsController.ts (hunk 3 above)
Comment (danadi7) [reply apps/api/src/ordinals/OrdinalsController.ts]: File removed. This should be on the base-api branch on the other PR.

File: apps/api/src/transactions/TransactionsController.ts (hunk 5 above)
Comment (danadi7) [reply apps/api/src/transactions/TransactionsController.ts]: File removed. This should be on the base-api branch on the other PR.

File: apps/api/src/ordinals/OrdinalsModule.ts (hunk 4 above)
Comment (danadi7) [reply apps/api/src/ordinals/OrdinalsModule.ts]: File removed. This should be on the base-api branch on the other PR.

File: apps/api/src/transactions/TransactionsModule.ts (hunk 6 above)
Comment (danadi7) [reply apps/api/src/transactions/TransactionsModule.ts]: File removed. This should be on the base-api branch on the other PR.

**Review by joundy — APPROVED** [review]

**Pull request merged**